posts/2021-10-12-open-beta-changes.md

```diff
@@ -233,16 +238,50 @@ listProducts( filter: { size: { eq: XL } } ) { Products { name sku price size }
 
-# Secure Message Feature Removal
+## Secure Message Feature Removal
 
 Vendia Share Unis are designed to make it easy to share data and code across companies, regions, and clouds. However, sometimes it's appropriate to restrict information to a subset of participants. Secure messaging was introduced to address this need. However, over the past year, we heard from customers that they would rather see this capability exposed when writing or updating data through standard GraphQL mutations. We introduced fine-grained data permissions to address this need. This capability allows data producers to limit which participants can take action on data written to a Uni.
 
-## Change Areas
+### Change Areas
 
 * **Removal of Secure Messaging** - We're updating Vendia Share to remove the Secure Messaging feature from the platform.
 
-## Change Areas Applied to Secure Messaging
+### Change Areas Applied to Secure Messaging
 
 We're removing secure messaging in favor of fine-grained data permissions. There are several advantages to using this capability. Secure messaging only allows for a message to be sent to one or more recipients; the data contained in the message is not written to the shared data model. It is not available to be read, updated, or deleted at a later point. In contrast, data written with fine-grained permission is stored in your model. As such, it is available to be read, updated, or deleted at a later point. In addition, permissions can be updated at any time to change the principal, path, or operations allowed on the data as your data sharing needs evolve.
 
 Please refer to our [blog post](https://www.vendia.net/blog/sharing-data-with-fine-grained-control) and [product documentation](https://www.vendia.net/docs/share/fine-grained-data-permissions) for how to make use of Share's fine grained data access controls.
+
+## Requiring Authorizer Type During Uni Creation
+
+Vendia Share Unis are created via either the [Command Line Interface](https://www.vendia.net/docs/share/cli) or the [Web Console](https://www.vendia.net/docs/share/quickstart/web).  In either case, a [registration.json](https://www.vendia.net/docs/share/cli/guide#format-of-the-registration-schema-and-initial-state-files) file provides initial information about the Uni.
+
+### Change Area
+
+* **Requiring authorizerType** - We've updated Vendia Share to require clients to explicitly set a value for authorizerType when creating a Uni.  This change eliminates the prior implicit default value o authorizerType to `API_KEY` when a Uni is created.  This will help ensure all users understand the authorizerType selected during Uni creation and will provide additional traceability back to the original `registration.json` file.
```

Comment:
```suggestion
* **Requiring authorizerType** - We've updated Vendia Share to require clients to explicitly set a value for authorizerType when creating a Uni.  This change eliminates the prior implicit default authorizerType value of `API_KEY` when a Uni is created.  This will help ensure all users understand the authorizerType selected during Uni creation and will provide additional traceability back to the original `registration.json` file.
```